website/archiver/listeners.py

```diff
@@ -0,0 +1,86 @@
+import celery
+import itertools
+
+from framework.tasks import handlers
+
+from website.archiver.tasks import (
+    archive,
```

Comment:
Any reason to not use `from website.archiver.tasks import archive` or `from website.archiver import tasks`?